include/firrtl.xml

```diff
@@ -16,6 +16,7 @@
       <item>is</item>
       <item>invalid</item>
       <item>inst</item>
+      <item>remote_inst</item>
```

Comment:
This is the only keyword that has an `_`. Types use UpperCamelCase and other keywords just elide any punctuation (eg. extmodule, intmodule, defname). Should we maybe make it `remoteinst` or `extinst` or something else?

Reply:
Changes to `remoteinst` in https://github.com/chipsalliance/firrtl-spec/pull/107/commits/a3654afff15b34b8c94f6a86868a8ece649262f9.